Cargo.toml

```diff
@@ -240,38 +242,53 @@ trace_tracy_memory = [
 # Tracing support
 trace = ["bevy_internal/trace"]
 
+# AVIF image format support
+avif = ["bevy_internal/avif"]
+
+# Basis Universal compressed texture support
+basis-universal = ["bevy_internal/basis-universal"]
+
+# BMP image format support
+bmp = ["bevy_internal/bmp"]
+
+# DDS compressed texture support
+dds = ["bevy_internal/dds"]
+
 # EXR image format support
 exr = ["bevy_internal/exr"]
 
+# Farbfeld image format support
+ff = ["bevy_internal/ff"]
+
 # HDR image format support
 hdr = ["bevy_internal/hdr"]
 
-# PNG image format support
-png = ["bevy_internal/png"]
+# KTX2 compressed texture support
+ktx2 = ["bevy_internal/ktx2"]
 
-# TGA image format support
-tga = ["bevy_internal/tga"]
+# ICO image format support
+ico = ["bevy_internal/ico"]
 
 # JPEG image format support
 jpeg = ["bevy_internal/jpeg"]
 
-# BMP image format support
-bmp = ["bevy_internal/bmp"]
+# PNG image format support
+png = ["bevy_internal/png"]
 
-# WebP image format support
-webp = ["bevy_internal/webp"]
+# PNM image format support, includes pam, pbm, pgm and ppm
+pnm = ["bevy_internal/pnm"]
 
-# Basis Universal compressed texture support
-basis-universal = ["bevy_internal/basis-universal"]
+# QOI image format support
+qoi = ["bevy_internal/qoi"]
 
-# DDS compressed texture support
-dds = ["bevy_internal/dds"]
+# TGA image format support
+tga = ["bevy_internal/tga"]
 
-# KTX2 compressed texture support
-ktx2 = ["bevy_internal/ktx2"]
+# TIFF image format support
+tiff = ["bevy_internal/tiff"]
 
-# PNM image format support, includes pam, pbm, pgm and ppm
-pnm = ["bevy_internal/pnm"]
+# WebP image format support
+webp = ["bevy_internal/webp"]
```

Comment:
These were sorted alphabetically since it seemed confusing otherwise. Like mentioned, `avif`, `ff`, `gif`, `ico`, `qoi`, and `tiff` are the new ones.